modules/fakemeta/fakemeta_amxx.cpp

```diff
@@ -13,12 +13,16 @@
 
 #include "fakemeta_amxx.h"
 #include "sh_stack.h"
+#include <reapi/mod_regamedll_api.h>
```

Comment:
I had this thought too but forgot about it. I'm okay with `resdk`, will change it soon.